spec/views/events/show.html.erb_spec.rb

```diff
@@ -64,6 +64,26 @@
     expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges'))
   end
 
+  it "should not display accept-all-button for non-organizers" do
+    sign_in(FactoryGirl.create(:user, role: :coach))
```

Comment:
They don't see it because they don't see the applicants overview in general, but it probably would be good to have it in the test explicitly, so I added it